### ISSUE TITLE
Show an indicator if a Jetpack site is unreachable

### ISSUE THREAD

**Issue by koke** [wordpress-mobile/WordPress-iOS#8641]
In calypso, when WordPress.com can't connect to a Jetpack site, there's an indicator to let the user know there's a problem, and has an option to disconnect the site.

![connection-indicator](https://user-images.githubusercontent.com/8739/36376475-8260b892-1573-11e8-9a2b-b9af5dbc9853.png)

We should have something similar in the app, as otherwise we silently fail when the site is down.

**Comment by elibud** [comment]
@mattmiklic can you come up with a design to address this on the apps? Thx

**Comment by SylvesterWilmott** [comment]
I think we can reuse what we have for the "Register Domain" alert on the dashboard. Explicit designs in "misc" in Zeplin.

<img width="429" alt="Screenshot 2019-07-30 at 10 21 53" src="https://user-images.githubusercontent.com/36532468/62117460-e1e0b180-b2b3-11e9-9142-6005b60fd441.png">

**Comment by SylvesterWilmott** [comment]
@designsimply I would say that in this case the design wasn't "accepted" as such and so the label was left there.

Pinging @mattmiklic

**Comment by koke** [comment]
That's definitely better than what we have and I'd be happy to ship that. That said, the layout isn't super clear to me as the error is detached from the icon and it seems easy to miss. On a first glance, I might just see a "Remove Site" option that happens to have a dangerous looking icon.

Perhaps we should just have the error in that screen, and when the user taps then offer to remove the site or contact support?

**Comment by designsimply** [comment]
@SylvesterWilmott can you please follow-up on the latest comments for us here? Especially since this issue has a `[Pri] High` label. Thank you!

**Comment by SylvesterWilmott** [comment]
We use a similar solution when informing the user that they have an unregistered premium domain where the row is the action and the footer text describes what it is.
<img width="408" alt="Screenshot 2020-03-09 at 10 26 27" src="https://user-images.githubusercontent.com/36532468/76204517-715a2580-61f0-11ea-871d-534f369dba66.png">

Perhaps, in this case, we can expand the footer text to be more descriptive. @koke Can you give a little more info about when a user receives this type of error.

**Comment by koke** [comment]
> @koke Can you give a little more info about when a user receives this type of error.

I think there are several reasons why WordPress.com might have trouble connecting, but perhaps the simplest to reproduce is when the site is down. I shut down nginx on my server and the error appeared on calypso.

**Comment by SylvesterWilmott** [comment]
If we have multiple causes I think the following design is good. It's also consistent with our other alert type.

<img width="405" alt="Screenshot 2020-03-09 at 11 05 47" src="https://user-images.githubusercontent.com/36532468/76207560-f0059180-61f5-11ea-88c0-5f15f831ba03.png">

**Comment by koke** [comment]
A user in 3418322-zen wanted to disconnect some sites that didn't exist anymore and wasn't able to

**Comment by yaelirub** [comment]
Wondering if this is related to https://github.com/wordpress-mobile/WordPress-iOS/issues/8641 ?
@designsimply wdyt?

**Comment by designsimply** [comment]
@yaelirub 🤨  I think you meant https://github.com/wordpress-mobile/WordPress-iOS/issues/13994 and, yes, I think we can close that one as a duplicate. I'll do that now. 👍

**Comment by sla8c** [comment]
I took a look at this issue + reviewed the context of #13994 as well.  

I think this ticket can be closed now.  In the JetPack and WordPress iOS app when the app can no longer connect to a WordPress site without the JetPack plugin, the site will be removed from the list of sites within the app

Steps to reproduce:
I created a Self hosted site (using jurassic ninja) with the JetPack plugin installed and activated
2. I added the site to the JetPack and WordPress iOS app(s) respectively.  This site correctly shows in the list of "My Sites"
3. I went to the web UI -> Plugins and I de-activated the Jetpack plugin
4. Navigating back to the WordPress and JetPack iOS apps, the site is dynamically removed when the My Sites table is updated

Looking closer / inspecting the iOS app code, I can see that in class `BlogListViewController` on the func `viewWillAppear`, syncBlogs will be called each time which does a fresh network call to get the fresh list of blogs for the user.  When this network call doesn't return the site that had the JetPack plugin de-activated it will automatically be removed when the tableview is refreshed without this blog in the API results.

@SylvesterWilmott if you agree that the above means we can close this issue I'm happy to close it.  Thanks!
